EngineLayer/CompactToProteinMatchEngine/SequencesToActualProteinPeptidesEngine.cs

```diff
@@ -74,20 +82,18 @@ protected override MetaMorpheusEngineResults RunSpecific()
             double proteinsMatched = 0;
             int oldPercentProgress = 0;
 
-            Parallel.ForEach(Partitioner.Create(0, proteins.Count), fff =>
-            {
-                for (int i = fff.Item1; i < fff.Item2; i++)
+            
```

Comment:
revert to parallelized, we should investigate if there's a bug in the parallelization code (it's super slow in some cases without parallelization)